pylabrobot/config/config.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Union
+
+LOG_FROM_STRING = {
+  "DEBUG": logging.DEBUG,
+  "INFO": logging.INFO,
+  "WARNING": logging.WARNING,
+  "ERROR": logging.ERROR,
+  "CRITICAL": logging.CRITICAL,
+}
+
+LOG_TO_STRING = {
+  logging.DEBUG: "DEBUG",
+  logging.INFO: "INFO",
+  logging.WARNING: "WARNING",
+  logging.ERROR: "ERROR",
+  logging.CRITICAL: "CRITICAL",
+}
```

Comment:
perhaps `LOG_TO_STRING = {v: k for k, v in LOG_FROM_STRING.items() }`?

Reply:
Yes that's a better idea, changed

Reply:
need to push? :)